k8s/tools/deploy-celery-gke.sh

```diff
@@ -0,0 +1,245 @@
+#!/bin/bash
+# Turbinia GKE deployment script.
+# This script can be used to deploy the Turbinia Celery stack to GKE. 
+# Requirements:
+# - have 'gcloud' and 'kubectl' installed.
+# - autheticate against your GCP project with "gcloud auth login"
+# - account being used to run script should have an IAM policy of instance.admin and container.admin used to create the necessary resources.
+# - optionally have the GCP project set with "gcloud config set project [you-project-name]"
+#
+# Use --help to show you commands supported.
+
+set -o posix
+set -e
+
+# Source cluster config to pull specs to create cluster from. Please review
+# the config file and make any necessary changes prior to executing this script
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source $DIR/.clusterconfig
+cd $DIR/..
+
+if [[ "$*" == *--help ]] ; then
```

Comment:
I know this is a carry-over from the other script, but could we print this out when the user specifies `-h` as well?

Reply:
done